AutomatticTracks/src/main/java/com/automattic/android/tracks/CrashLogging/CrashLogging.kt

```diff
@@ -0,0 +1,104 @@
+package com.automattic.android.tracks.crashlogging
+
+import android.content.Context
+import com.automattic.android.tracks.BuildConfig
+import io.sentry.*
+import io.sentry.android.core.SentryAndroid
+import io.sentry.protocol.Message
+import io.sentry.protocol.User
+import java.util.*
+
+object CrashLogging {
+
+    private const val EMPTY_VALUE = ""
+
+    private lateinit var dataProvider: CrashLoggingDataProvider
+
+    @JvmStatic
+    fun start(
+            context: Context,
+            dataProvider: CrashLoggingDataProvider,
+    ) {
+        SentryAndroid.init(context) { options ->
+            options.apply {
+                dsn = dataProvider.sentryDSN
+                environment = dataProvider.buildType
+                release = dataProvider.releaseName
+                setDebug(BuildConfig.DEBUG)
+                setLogger(if (BuildConfig.DEBUG) SystemOutLogger() else NoOpLogger.getInstance())
+                setTag("locale", getCurrentLanguage(dataProvider.locale))
+                beforeSend = SentryOptions.BeforeSendCallback { event, _ ->
+                    return@BeforeSendCallback if (dataProvider.userHasOptedOut) {
```

Comment:
Here's also change I'd like to border. Previously we didn't send event if library was in `debug` variant but I'd like to leave this decision only to client. There are cases when we want to send event in `debug` (testing)